sasmodels/resolution2d.py

```diff
@@ -181,50 +181,55 @@ class Slit2D(Resolution):
 
     *q* points at which the data is measured.
 
-    *qx_width* slit width in qx
+    *q_length* slit length (long axis); assumed to be in the direction of qx
 
-    *qy_width* slit height in qy; current implementation requires a fixed
-    qy_width for all q points.
+    *q_width* slit width (short axis); assumed to be in the direction of qy; current implementation requires a fixed
+    q_width for all q points.
+
+    Please note that this assumption of laboratory-frame qx and qy directions can be inverted by adding or subtracting
+    90 degrees from the model orientation.  For the particular case of USANS, which has a vertical slit of width
+    *q_width* sweeping through qx, add 90 degrees to the fitted phi angle to find the orientation relative to laboratory
+    frame.
```

Comment:
I suspect only one of added or subtracted is correct but we will only be able to see that with asymmetric scattering patterns. Would we get that from something that breaks symmetry about *a-b*, *a-c* or *b-c*, like an egg or a pyramid? This should be easy enough to test with realspace.py. Pringle doesn't help since forward and backward rotation by 90º should be the same wrt slit integration.

Reply:
except for reflections scattering should always have a center of inversion symmetry so that $x$ and $x + pi$  should be identical as far as I know.